python-backend/math_utils.py

```diff
@@ -0,0 +1,36 @@
+import sympy
+from sympy import Symbol
+
+import constants
+
+GRAPHABLE_TYPES = (int, float, sympy.core.numbers.Integer, sympy.core.numbers.Float)
+PRECISION = constants.PRECISION
+X_VALUES = range(0, 1000, 25)
+
+
+def error_coordinates(expression: sympy.core, symbol: Symbol, limit: sympy.core.numbers):
```

Comment:
Provide the return type. Please do this for all python functions that you write.

Reply:
I always forget. I'll see if I can enable a check to remind me to do this...